src/components/CreditCard.jsx

```diff
@@ -0,0 +1,73 @@
+import Visa from '../assets/images/Visa.png';
```

Comment:
```suggestion
import Visa from '../assets/images/visa.png';
```
La has puesto en assets con minuscula, que si no da error!! :D